docs/src/examples/lorenz_example.md

```diff
@@ -1,26 +1,289 @@
+
+Lorenz dynamics settings: The use of the transient forcing term is with the flag, `dynamics`. Stationary forcing is `dynamics=1` ($A=0$) and transient forcing is used with `dynamics=2` ($A\neq0$).
+The default parameters are specified in `calibrate.jl` and can be modified as necessary.
+The system is solved over time horizon $0$ to `tend` at fixed time step `dt`.
+```julia
+N = 36
+dt = 1/64
+t_start = 800
+```
+
+Inverse problem settings: The states are integrated over time `Ts_days` to construct the time averaged statistics for use by the optimization.
+The specification of the statistics to be gathered from the states are provided by `stats_type`.
+The Ensemble Kalman Process (EKP) settings are
+```julia
+N_ens = 20 # number of ensemble members
+N_iter = 5 # number of EKI iterations
+```
+
+Setting up the Inverse Problem: The goal is to learn ```F_s``` and ```A``` based on the time averaged statistics in a perfect model setting.
+The true parameters are
+```julia
+F_true = 8. # Mean F
+A_true = 2.5 # Transient F amplitude
+ω_true = 2. * π / (360. / τc) # Frequency of the transient F
+params_true = [F_true, A_true]
+param_names = ["F", "A"]
+```
+
+Priors: We implement (biased) priors as follows
+
+```julia
+prior_means = [F_true + 1.0, A_true + 0.5]
+prior_stds = [2.0, 0.5 * A_true]
+# constrained_gaussian("name", desired_mean, desired_std, lower_bd, upper_bd)
+prior_F = constrained_gaussian(param_names[1], prior_means[1], prior_stds[1], 0, Inf)
+prior_A = constrained_gaussian(param_names[2], prior_means[2], prior_stds[2], 0, Inf)
+priors = combine_distributions([prior_F, prior_A])
+```
+We use the recommended [`constrained_gaussian`] to add the desired scale and bounds to the prior distribution, in particular we place lower bounds to preserve positivity. 
+
+The priors can be plotted directly using `plot(priors)`, as seen below in the example code from `calibrate.jl`
+```julia
+# Plot the prior distribution
+p = plot(priors, title = "prior")
+plot!(p.subplots[1], [F_true], seriestype = "vline", w = 1.5, c = :steelblue, ls = :dash, xlabel = "F") # vline on top histogram
+plot!(p.subplots[2], [A_true], seriestype = "vline", w = 1.5, c = :steelblue, ls = :dash, xlabel = "A") # vline on top histogram
+figpath = joinpath(figure_save_directory, "prior" * ".pdf")
+savefig(figpath)
+figpath = joinpath(figure_save_directory, "prior" * ".png")
+savefig(figpath)
+```
+
+```@raw html
+<img src="../../assets/Lorenz-prior.png" width="600">
+```
+
+Observational Noise: The observational noise can be generated using the L96 system or prescribed, as specified by `var_prescribe`. 
+
+`var_prescribe==false`
+The observational noise is constructed by generating independent instantiations of the L96 statistics of interest at the true parameters for different initial conditions.
+The empirical covariance matrix is constructed.
+
+`var_prescribe==true`
+The observational noise is prescribed as a Gaussian distribution with prescribed mean and variance.
+
```

Comment:
Right now you define the prior, but then it jumps to the emulation.

Could you 
- mention EKI here as the way to solve the problem (maybe with a snippet if you like but don't need as much detail) - point to e.g. EnsembleKalmanProcesses.jl
- add the calibration results here instead of at the end.

Then move on with the E-S stuff.